### PR TITLE
Add environment variable support

### LINKED ISSUE #10: source Atlas public and private key from env
☝🏼

### PR DESCRIPTION
**Pull request by n3ph**
☝🏼 

- resolves https://github.com/maoertel/mongodb-ftdc/issues/10

### CODE REVIEW

**Comment by n3ph** [comment]
I think the variables in `service.rs` are way too generic:
https://github.com/maoertel/mongodb-ftdc/blob/d3271703d11813a321f0faed71653fe9f0009081/src/service.rs#L26-L27
But didn't want to bloat the PR for this.

File: src/cli.rs [hunk 1]
```diff
@@ -22,9 +22,9 @@ pub(crate) struct Cli {
...
     /// The public key of your Atlas API key.
-    #[clap(long)]
-    pub(crate) public: String,
+    #[clap(long, env)]
+    pub(crate) atlas_public_key: String, // --atlas-public-key or ATLAS_PUBLIC_KEY env var
     /// The private key of your Atlas API key.
-    #[clap(long)]
-    pub(crate) private: String,
+    #[clap(long, env)]
+    pub(crate) atlas_private_key: String, // --atlas-private-key or ATLAS_PRIVATE_KEY env var
```
Comment (maoertel) [inline src/cli.rs]: Wdyt about not adding comments but rather make it more explicit code wise, like this:

```rust
    #[clap(long, env = "ATLAS_PRIVATE_KEY")]
    pub(crate) atlas_private_key: String,
```

**Comment by maoertel** [comment]
> I think the variables in `service.rs` are way too generic:
> 
> https://github.com/maoertel/mongodb-ftdc/blob/d3271703d11813a321f0faed71653fe9f0009081/src/service.rs#L26-L27
> 
> But didn't want to bloat the PR for this.

This is fine, as this is a binary crate as well as a lib crate this means that this changes the public API, so it will not be a patch updade. Just to keep in mind.

File: src/cli.rs (hunk 1 above)
Comment (n3ph) [reply src/cli.rs]: Done with https://github.com/maoertel/mongodb-ftdc/pull/11/commits/90df26fac894dca995a457a100fee7f69b67a9b9

**Review by maoertel — APPROVED** [review]

**Pull request merged**